aiohttp_traversal/ext/views.py

```diff
@@ -8,7 +8,8 @@
 
 
 class View(AbstractView):
-    def __init__(self, request, resource, tail):
+    def __init__(self, request, resource, tail, serializer=None):
+        self.serializer = serializer or json.dumps
```

Comment:
I think `json.dumps` should be placed as serializer default. A bit more explicit to notice.